en/components/getting_started.md

```diff
@@ -0,0 +1,299 @@
+---
+title: Getting Started
+_description: Ignite UI for Angular is a complete set of native Angular UI components, which allow you to build fast, feature-rich and Material designed applications!
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library
+---
+
+## Getting started with Ignite UI for Angular
+
+[`Ignite UI for Angular`](https://github.com/IgniteUI/igniteui-angular) is a complete set of Material-based UI Widgets, Components & Sketch UI kits and supporting directives for Angular by Infragistics. Ignite UI for Angular is designed to enable developers to build the most modern, high-performance HTML5 & JavaScript apps for modern desktop browsers, mobile experiences and progressive web apps (PWA’s) targeting Google's Angular framework.
+
+Before you can run Ignite UI for Angular, there are 2 steps to get everything on your machine set up to run any Angular app, including Ignite UI for Angular, and to build Ignite UI for Angular apps.
+
+To get started, you must install both NodeJS and Visual Studio Code on your machine. Modern web development with Angular requires NodeJS, it’s sort of like an ASP.NET app running in the browser and requires ASP.NET runtime deployed on the server. For development, there is nothing better than Visual Studio Code to build Angular apps. Getting started is easy, just download and install both Node.js and Visual Studio Code.
+
+<div>
+    <div style="display:inline-block;width:45%;text-align:center;">
+      <img src="../images/general/nodejs.svg"
+           style="display:flex;max-height:100px;margin:auto auto 20px auto;" />
+      <a target="_blank" href="https://nodejs.org/en/download/"
+         style="color:white;background-color:#09f;text-decoration:none;font-weight:700;font-size:16px;padding: 5px 15px 5px 15px;">
+        DOWNLOAD NODE
+      </a>
+    </div>
+    <div style="display:inline-block;width:45%;text-align:center;">
+      <img src="../images/general/vs-code.svg"
+           style="display:flex;max-height:100px;margin:auto auto 20px auto;" />
+      <a target="_blank" href="https://code.visualstudio.com/download"
+         style="color:white;background-color:#09f;text-decoration:none;font-weight:700;font-size:16px;padding: 5px 15px 5px 15px;">
+        DOWNLOAD VS CODE
+      </a>
+    </div>
+</div>
+<div class="divider--half"></div>
+
+## Installing Ignite UI for Angular
+
+Now that we have everything we need, we can start creating/updating our application!
+
```

Comment:
The general guidance for the users should be with the following order: 

1. create project with the igniteui-cli -> Use the ignteui-angular
2. Update your existing angular application -> Use ng add igniteui-angular -> use the igniteui-cli, or add manually components.
3. Create application using ng cli -> use ng add igniteui-angular
@kdinev - I vote to remove number 3. , because if they choose to use ng new, they will fall into number 2. Your opinion?